aptos-move/aptos-vm/src/move_vm_ext/code_context.rs

```diff
@@ -143,6 +145,11 @@ fn native_request_publish(
     mut args: VecDeque<Value>,
 ) -> PartialVMResult<NativeResult> {
     debug_assert_eq!(args.len(), 4);
+
+    if !cfg!(any(test, feature = "fuzzing")) {
```

Comment:
What is feature = "fuzzing" for?

Reply:
Fuzzing is not on in production builds. This was a recommendation from @davidiw  how to turn this off.